CharacterCreation/Class-Specific Documentation/Doctor/Doctor Hippocratic Proceedures.txt

```diff
@@ -0,0 +1,337 @@
+==============================
+Hippocratic Proceedures
+==============================
+
+==Patch Em Up ==
+Adrenaline Reward: 40 Points
+Contagion Reward: 1 Contagion 
+Range:          Touch
+Action:         2 Actions
+    
+    You use your biofoam appllicator to spray the smelly green foam into their
+wounds. The miracle of modern science rapidly reconnects arteries and veins and
+begins to mimic destroyed organs and muscle tissue. It hurts like hell, but it
+will give them a chance to keep it together long enough to get a more permanent
+solution back at base. 
+
+    You must have your Field Applicator drawn in order to use this ability. Your
+patient heals an ammount of health listed on the table below. As your character
+learns more medicine skill, the better they heal. Unless otherwise stated, this
+ability cannot heal a patient above half health.
+
++-----------------+--------------+--------+
+| Med Skill Above | Base Healing |  Dice  |
++-----------------+--------------+--------+
+|       9         |     20       | + 2d10 |
++-----------------+--------------+--------+
+|       19        |     30       | + 3d10 |
++-----------------+--------------+--------+
+|       29        |     35       | + 4d10 |
++-----------------+--------------+--------+
+|       39        |     40       | + 5d10 |
++-----------------+--------------+--------+
+|       49        |     50       | + 6d10 |
++-----------------+--------------+--------+
```

Comment:
Table's great.
For reference, average heal at each step is 31, 46, 57, 67, 83. With that interesting new half-health healing limit in mind, not a lot of PCs are going to have 160 health, let alone 220 (max roll). Thoughts on reducing that 50 to a 45 and adding a D10 every other level instead of every?

EDIT: I found Pre-Op Triage below. Still interested in the above-mentioned thoughts.

Reply:
I think I want to make sure that the doctor is rolling a whole handful of dice, and basically always gets a mathematically successful heal. I think it feels good for the doctor player, and tactically, I think it's important that the healers are effective healers and don't have the chance to have perfect footwork, but fail to heal. That wouldn't provide a good tactical challenge.